docs/concept/buckconfig.soy

```diff
@@ -1570,16 +1570,34 @@ $ buck targets --resolve-alias app#src_jar
   {param name: 'jdk_name' /}
   {param example_value: 'Java SDK 1.6' /}
   {param description}
-    IntelliJ project SDK name.
+    IntelliJ project Java SDK name.
```

Comment:
`jdk_name` is a bad name here, it's an SDK name, not JDK name. It's better leave it as it was before (i.e. just SDK name) because it can point to an Android SDK or Intellij Plugin SDK.